core/domain/activity_jobs_one_off.py

```diff
@@ -673,6 +673,15 @@ class AddMissingCommitLogsJob(jobs.BaseMapReduceOneOffJobManager):
 
         }
     }
+    # "Public" is the default value for post_commit_status for the
```

Comment:
Add a backend test verifying that the two lists below are distinct and that their union is equal to the set of keys in the dict above.

Reply:
Done.

Reply:
Sorry ... I don't see it?

Reply:
Since the list had only one item, I decided to remove the list and thus removed the test. But now I have added it back and added the test.